src/components/Banner/AnonBanner/AnonBanner.module.scss

```diff
@@ -0,0 +1,103 @@
+@import '../../../../node_modules/@santiment-network/ui/variables';
+@import '../../../mixins';
+
+.banner {
```

Comment:
Current styles do not match base styles in [master](https://github.com/santiment/app/blob/a5ea90525cc3f631177f5641646c699d80ebde0b/src/components/AnonBanner/AnonBanner.module.scss#L1)

Reply:
yes, because Yura requested make static banner adaptive for tablet and mobile

Reply:
@haritonasty I'm not speaking about responsive mixins. `__top`, `__description` and so on differ from the master branch, which has a clear separation between different variants

Reply:
ok, done)